Marlin/language_de.h

```diff
@@ -36,15 +36,20 @@
 #define WELCOME_MSG                         MACHINE_NAME " bereit"
 #define MSG_SD_INSERTED                     "SD-Karte erkannt"
 #define MSG_SD_REMOVED                      "SD-Karte entfernt"
+#define MSG_LCD_ENDSTOPS                    "Endstopp" // Max length 8 characters
 #define MSG_MAIN                            "Hauptmenü"
 #define MSG_AUTOSTART                       "Autostart"
-#define MSG_DISABLE_STEPPERS                "Motoren aus" // M84
+#define MSG_DISABLE_STEPPERS                "Motoren deaktivieren" // M84
 #define MSG_AUTO_HOME                       "Home" // G28
+#define MSG_AUTO_HOME_X                     "Home X"
+#define MSG_AUTO_HOME_Y                     "Home Y"
+#define MSG_AUTO_HOME_Z                     "Home Z"
 #define MSG_LEVEL_BED_HOMING                "Homing XYZ"
 #define MSG_LEVEL_BED_WAITING               "Klick für Start"
+#define MSG_LEVEL_BED_NEXT_POINT            "Nächste Koordinate"
 #define MSG_LEVEL_BED_DONE                  "Fertig"
 #define MSG_LEVEL_BED_CANCEL                "Abbruch"
-#define MSG_SET_HOME_OFFSETS                "Setze Homeoffsets"
+#define MSG_SET_HOME_OFFSETS                "Setze Homeversatz"
```

Comment:
Even though we agreed to change this, I would hold out on this change for now, since we will have to replace a few instances of "offset" when we change this. Other than that it looks good to me.